tests/test_core.py

```diff
@@ -880,6 +881,22 @@ def test_delimited_list_as_string(web_request, parser):
     assert data["ids"] == "1,2,3"
 
 
+def test_delimited_list_as_string_v2(web_request, parser):
+	# b26793ee8021b6cfe525ac4f05570c3c68a4debd commit info
```

Comment:
Nitpick: No need to include the commit info here.

Reply:
🤣，I just want to others know why I add this test case

Reply:
That's what `git blame` is for =)

Reply:
ok, i will delete this line

Reply:
why CI cannot pass?

Reply:
See my comment above. You need to format the code using `invoke precommit`.

Reply:
Let me know if you have trouble getting the command to work. I can help you work through the errors if you want.

Or, if you don't have time, I'd be glad to make the change myself.

Reply:
I very appreciate your kindness. I do have some troubles in this command. I do not know how to execute this command. So, could you tell the steps to work it?

Reply:
Thank you very much, I never make a pr before

Reply:
That's great! Thank *you* for taking the time to make the PR.